docs/assets/includes/api-token-warning.md

```diff
@@ -1,4 +1,4 @@
 !!! warning
     **Never write API tokens to your configuration files** and keep your API tokens well protected, as they grant owner permissions to your projects on Codacy.
 
-    We recommend that you set API tokens as environment variables. Check the documentation of your CI/CD platform on how to do this.
+    It's a best practice to store API tokens as environment variables. Check the documentation of your CI/CD platform on how to do this.
```

Comment:
I'd even go this far. Overly prescriptive?

```suggestion
    Always store API tokens as environment variables. Check the documentation of your CI/CD platform on how to do this.
```

Reply:
We already tell users to "**never write** API tokens [...]", so here I believe it makes more sense to present the solution of using environment variables as a best practice. Also, depending on the CI/CD platform, the actual solution to secure the API tokens may be different.